<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsDialog</class>
 <widget class="QDialog" name="DetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Details</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="icon">
       <iconset resource="../images.qrc">
        <normaloff>:/images/blockdevice.png</normaloff>:/images/blockdevice.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="1" column="0">
        <layout class="QGridLayout" name="statusHLayout">
         <property name="margin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="doneGLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="doneIndic">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../images.qrc">:/images/graph-downloaded.png</pixmap>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="doneLabel">
             <property name="text">
              <string>Done</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="activeGLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="activeIndic">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../images.qrc">:/images/graph-downloading.png</pixmap>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="activeLabel">
             <property name="text">
              <string>Active</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QGridLayout" name="outstandingGLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="outstandingIndic">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../images.qrc">:/images/graph-notdownload.png</pixmap>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="outstandingLabel">
             <property name="text">
              <string>Outstanding</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <spacer name="statusHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>368</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="needCheckingGLayout">
           <item>
            <widget class="QLabel" name="needCheckingIndic">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../images.qrc">:/images/graph-checking.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="needCheckingLabel">
             <property name="text">
              <string>Needs checking</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QScrollArea" name="fileTransferInfoWidget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>747</width>
            <height>509</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLinks">
      <attribute name="title">
       <string>retroshare link(s)</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="linksGBox">
         <property name="title">
          <string>retroshare link</string>
         </property>
         <layout class="QGridLayout">
          <item row="1" column="0">
           <widget class="QTextEdit" name="linkText"/>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="copylinkdetailsButton">
            <property name="text">
             <string>Copy link to clipboard</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetailsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
